packages/react/src/components/TableHandles/TableHandleMenu/DefaultButtons/AddButton.tsx

```diff
@@ -84,8 +86,14 @@ export const AddColumnButton = <
           type: "tableContent",
           columnWidths,
           rows: props.block.content.rows.map((row) => {
-            const cells = [...row.cells];
-            cells.splice(props.index + (props.side === "right" ? 1 : 0), 0, []);
+            const cells = [...row.cells] as
```

Comment:
the addcolumn button breaks when there's a cell with multiple rowspans. Let's see if and how prosemirror-tables handles this?

![alt text](image.png)

after:

![alt text](image.png)